.circleci/config.yml

```diff
@@ -128,6 +138,23 @@ aliases:
       name: Waiting for Dockerized request replayer
       command: wait-for request-replayer:80 --timeout=120
 
+  - &STEP_WAIT_TEST_AGENT
+    run:
+      name: Waiting for Dockerized APM Test Agent
+      command: wait-for test-agent:9126 --timeout=120
+
+  - &STEP_DOCKER_LOGS
+    run:
+      name: Outputting Docker Logs for APM Test Agent
+      command: docker logs -f test-agent
+      background: true
+
+  - &STEP_RUN_TEST_AGENT_SPAN_VALIDATION
+    run:
+      name: Getting APM Test Agent Span Validation Results
```

Comment:
Do you have more details/doc on this one?

Reply:
removing, future work